### PR TITLE
cpu max usage + docs + bundle update

### PR DESCRIPTION
**Pull request by FarajiA**
## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-50

## Description:
Add CPU Max Usage graph component

## Testing:
Open /docs/index.html and navigate to cpu max usage

## Screenshots:
(if applicable)

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -22,6 +22,19 @@ export default [
 			commonjs() // so Rollup can convert any dependencies to an ES module
 		]
 	},
+	{
+		input: 'index.js',
```
Comment (FarajiA) [inline rollup.config.js]: This will populate the umd js file to the docs scripts folder.

File: src/components/cpu/cpu-max-usage/cpu-max-usage.js [hunk 2]
```diff
@@ -18,110 +18,66 @@ export class CpuMaxUsage extends HTMLElement {
...
-        var value = this.attributes.getNamedItem('data').value;
-        var data = JSON.parse(value);
-        this.marshalledData = [];
-
-        for (let index in data[0].valuesCollection) {
-            this.marshalledData.push({ time: data[0].valuesCollection[index][0], cpu_max_usage: data[0].valuesCollection[index][8] });
-        }
-
-        return this
+    static get observedAttributes() {
```
Comment (FarajiA) [inline src/components/cpu/cpu-max-usage/cpu-max-usage.js]: I'm thinking we can have this component make use of the observedAttributes to listen for changes. After some testing this seems to be the route that works best for 2 way databinding frameworks like Angular & React.

Comment (kyleondata) [reply src/components/cpu/cpu-max-usage/cpu-max-usage.js]: One thing to keep in mind here is order of operations.  In my testing, which you may have a way around this, I have observed that the observedChangesCallback fires before the connected callback, which has interesting/unintended effects.  Let's sync up on this.

Comment (kyleondata) [reply src/components/cpu/cpu-max-usage/cpu-max-usage.js]: after a zoom review everything looks good

**Pull request merged**